athena-shortcodes.php

```diff
@@ -35,8 +35,9 @@ function athena_sc_get_cache_bust() {
 	}
 }
 
-define( 'ATHENA_SC__CACHE_BUST', athena_sc_get_cache_bust() );
-
+add_action('init', function() {
+    define( 'ATHENA_SC__CACHE_BUST', athena_sc_get_cache_bust() );
+});
```

Comment:
So, this will probably work most of the time, but there is a chance that this function will get called after the `athena_sc_tinymce_init` function - which is also on the `init` action - which will then cause a more severe error, as the constant won't be defined.

What I would recommend doing instead is just removing the constant entirely and replacing it with the function call in the one place it's used:

So on line 118:

```php
add_editor_style( plugins_url( 'static/css/athena-editor-styles.min.css?' . athena_sc_get_cache_bust(), ATHENA_SC__PLUGIN_FILE ) );
```

Reply:
I would have to assume we thought we were going to need to use that constant in more than just this one place, and then never ended up doing so. Since it's only being used one place to cache bust the stylesheet, I think it's fine to just get rid of the constant.

Reply:
Thank you for the details, I have removed the constant and replaced it with the fuction on line 118